cmd/config/parse.go

```diff
@@ -28,6 +30,16 @@ type OneBuildConfiguration struct {
 	Commands []map[string]string `yaml:"commands"`
 }
 
+// getConfigFile returns the 1build configuration file from root file flag or global file variable
```

Comment:
Should we put this function in `io.go`

as no use in this file & io has all its uses.